setup.cfg

```diff
@@ -1,2 +1,7 @@
 [pep257]
 ignore = D100,D203
+match = ((?!test_)(?!tube\.py).)*\.py
```

Comment:
I realized this wouldn't be needed if testtube's config file was named `test_tube.py` (but that would be a bit of a misnomer for the sake of hijacking pep257's default match config)